Ginger/GingerCoreNETUnitTest/RunTestslib/CLITest.cs

```diff
@@ -154,15 +169,19 @@ public void CLIHelp()
                 mConsoleMessages.Clear();
                 CLIProcessor CLI = new CLIProcessor();
 
-                // Act            
-                CLI.ExecuteArgs(new string[] { "help" });
+                // Act         
+                 Task.Run(async () =>
+                {
+                    await CLI.ExecuteArgs(new string[] { "help" });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5632404)